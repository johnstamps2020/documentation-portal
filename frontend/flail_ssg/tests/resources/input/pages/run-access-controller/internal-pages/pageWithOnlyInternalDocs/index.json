{
  "$schema": "/frontend/page-schema.json",
  "title": "Page with only internal docs",
  "template": "page",
  "class": "twoColumns",
  "items": [
    {
      "label": "Release Notes",
      "class": "subject releaseNotes",
      "items": [
        {
          "label": "Guidewire Cloud Platform Release Notes",
          "id": "supersecretgwcloudplatformreleasenotes"
        }
      ]
    },
    {
      "label": "Getting Started",
      "class": "subject featuresAndFunctionality",
      "items": [
        {
          "label": "Working with Guidewire Cloud Platform",
          "id": "supersecretcloudplatform"
        }
      ]
    },
    {
      "label": "Authentication",
      "class": "subject configuration",
      "items": [
        {
          "label": "Authentication with Guidewire Identity Federation Hub",
          "id": "supersecretgwidentityfederationhub"
        }
      ]
    }
  ]
}
