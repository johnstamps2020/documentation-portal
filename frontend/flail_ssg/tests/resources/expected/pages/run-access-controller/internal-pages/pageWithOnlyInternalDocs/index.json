{
  "$schema": "/frontend/page-schema.json",
  "title": "Page with only internal docs",
  "template": "page",
  "class": "twoColumns",
  "items": [
    {
      "label": "Release Notes",
      "class": "subject releaseNotes",
      "items": [
        {
          "label": "Guidewire Cloud Platform Release Notes",
          "id": "supersecretgwcloudplatformreleasenotes",
          "public": false,
          "internal": true
        }
      ]
    },
    {
      "label": "Getting Started",
      "class": "subject featuresAndFunctionality",
      "items": [
        {
          "label": "Working with Guidewire Cloud Platform",
          "id": "supersecretcloudplatform",
          "public": false,
          "internal": true
        }
      ]
    },
    {
      "label": "Authentication",
      "class": "subject configuration",
      "items": [
        {
          "label": "Authentication with Guidewire Identity Federation Hub",
          "id": "supersecretgwidentityfederationhub",
          "public": false,
          "internal": true
        }
      ]
    }
  ],
  "public": false,
  "internal": true
}
